### ISSUE TITLE
Validation split in the training time, Possible?

### ISSUE THREAD

**Issue by Lincoln93** [matterport/Mask_RCNN#2114]
In classification problems (especially in keras), we can split the training dataset into the train set and validation set using `validation_split` argument. In this mask-rcnn implement, is it possible to split the data in this manner?

we have two datasets (train, test) and prepare it. To make a validation split we can do this while preparing the data and end up with (train, val, test) OR can we do it after preparing the data, like split the train set into two parts (training part, and validation part)? Thanks.

**Comment by gizemtanriver** [comment]
@Lincoln93 hi, did you manage to implement validation split as you suggested? I am thinking of the same and not sure how to proceed..Thanks

**Comment by Lincoln93** [comment]
@gizemtanriver nope, `validation_split` can't be supported. We should create our validation set separately. The way `validation_spit` works randomly split the data set is not recommend. You can [check this](https://www.kaggle.com/ipythonx/gwd-with-mask-rcnn-and-coco-transfer-learning) for better demonstration.
